synapse/telepath.py

```diff
@@ -135,25 +135,24 @@ def mergeAhaInfo(info0, info1):
 
     return info0
 
-async def open(url, timeout=None):
+async def open(url, onlink=None):
     '''
-    Open a new telepath Client (or AHA Service Pool) based on the given URL.
-    '''
-    # backward compatible support for a list of URLs or urlinfo dicts...
-    if isinstance(url, (tuple, list)): # pragma: no cover
-        return await Client.anit(url)
-
-    urlinfo = chopurl(url)
-
-    if urlinfo.get('scheme') == 'aha':
+    Open a new telepath ClientV2 object based on the given URL.
 
-        ahaclient, ahasvc = await _getAhaSvc(urlinfo, timeout=timeout)
+    Args:
+        url (str): The URL to connect to.
```

Comment:
```suggestion
        url (str): The URL(s) to connect to.
```

Reply:
We may want to make this feature remain a bit obscure. I'm fine either way.

Reply:
Thin leaving it as `str` for now is best. We can expand it in the future.